Wmp checker remove sapstartsrv

nick-wang opened the pull request:
bsc#1184865, sapstartsrv is removed from process tree.
Enable SLE15SP3
Call `wmp-check` in support config.

Compare to https://github.com/SUSE/sapwmp/pull/10 , only the last two commits are new created. I will do the rebase to ease the review  after https://github.com/SUSE/sapwmp/pull/10 merged.
(Didn't find a way to include this two commits only, since `wmp-check` only existed in https://github.com/SUSE/sapwmp/pull/10)

nick-wang committed:
bsc#1184865, remove sapstartsrv from process tree and enable SLE15SP3 wmp-check.

nick-wang committed:
Call sapwmp check in supportconfig.

scmschmidt reviewed (commented):
Looks good to me.

Werkov reviewed (approved):
Also looks good to me (gave it a bit more cursory look, assuming it's tested).
I'd suggest adding `bsc#1184865` to commit (b4e4db94ed40086408acfb769d9e22a6cc37a731) message so that the reference pops up in changelog.

nick-wang commented:
Rebased based on the latest `main`

> Also looks good to me (gave it a bit more cursory look, assuming it's tested).

I did some basic test (sle15sp2) days ago. Will test with the final change.

> I'd suggest adding `bsc#1184865` to commit ([b4e4db9](https://github.com/SUSE/sapwmp/commit/b4e4db94ed40086408acfb769d9e22a6cc37a731)) message so that the reference pops up in changelog.

Good idea. I was thinking manually add this in `.changes`. Fixed in the rebased commit log, it is better now!

nick-wang commented:
Rebased based on the latest `main`

> Also looks good to me (gave it a bit more cursory look, assuming it's tested).
I did some basic test (sle15sp2) days ago. Will test with the final change.

> I'd suggest adding `bsc#1184865` to commit ([b4e4db9](https://github.com/SUSE/sapwmp/commit/b4e4db94ed40086408acfb769d9e22a6cc37a731)) message so that the reference pops up in changelog.
Good idea. I was thinking manually add this in `.changes`. Fixed in the rebased commit log, it is better now!
